documents/Libraries/sxpbrlib/sx-glsl/sx_addbsdf.glsl

```diff
@@ -0,0 +1,14 @@
+void sx_addbsdf_reflection(vec3 L, vec3 V, BSDF in1, BSDF in2, out BSDF result)
```

Comment:
L and V are not used. Do they need to be part of the signature? Same for V's below.

Reply:
Yes, it's not used for this BSDF node but it's used on most other BSDF node. The handling of calls to BSDF functions is currently static so it can't be changed from node to node. It's something we might be able to improve on in the future.